### ISSUE TITLE
Enable xUnit parallelization in tests

### ISSUE THREAD

**Issue by mthalman** [microsoft/dotnet-framework-docker#407]
The xUnit test framework has the capability to execute tests in parallel as long as they are in separate test classes.  Currently all the tests are in one class so they execute sequentially.  Experimentation has found that by executing the tests in parallel, we can improve test execution time by about 20%.

Executing the tests in parallel surfaces a few issues:
* A [Windows container bug](https://github.com/moby/moby/issues/37562) can occur when deleting a container.  Need to see whether this can be worked around by locking around stopping/removing a container.
* https://github.com/microsoft/dotnet-framework-docker/issues/400 occurs more frequently

**Comment by MichaelSimons** [comment]
We should consider doing this for the .NET Core repo as well.  @mthalman, can you log an issue for that?

**Comment by MichaelSimons** [comment]
This is currently blocked by #400

**Comment by mthalman** [comment]
This issue is being close as won't fix due to an inability to workaround the [Windows container bug](https://github.com/moby/moby/issues/37562) without adversely affecting the performance and thereby losing the benefits hoped to be gained by the parallelization.
